check-ecs-exec.sh

```diff
@@ -65,6 +65,7 @@ TASK_ID=${2:-None} # A task ID or a full ARN of the task
 if [[ "${CLUSTER_NAME}" = "None" || "${TASK_ID}" = "None" ]]; then
   printf "${COLOR_RED}Usage:\n" >&2
   printf "  ./check-ecs-exec.sh YOUR_ECS_CLUSTER_NAME YOUR_ECS_TASK_ID\n" >&2
```

Comment:
Could you possibly add `${COLOR_DEFAULT}` at the end of this line instead of adding new `printf` line? (and for the rest of changes as well)

```suggestion
  printf "  ./check-ecs-exec.sh YOUR_ECS_CLUSTER_NAME YOUR_ECS_TASK_ID${COLOR_DEFAULT}\n" >&2
```